.github/workflows/rust.yml

```
@@ -54,16 +54,16 @@ jobs:
           # aarch64 is disabled; because there is no native runner
 
           # macOS Targets - std features
-          - { os: macos-13, target: x86_64-apple-darwin, use-pgo: true, use-cross: false, features: "std mmap" }
-          - { os: macos-13, target: x86_64-apple-darwin, use-pgo: true, use-cross: false, features: "std mmap no-format" }
-          - { os: macos-13, target: x86_64-apple-darwin, use-pgo: true, use-cross: false, features: "std mmap no-format trim-file-lengths" }
+          - { os: macos-15-intel, target: x86_64-apple-darwin, use-pgo: true, use-cross: false, features: "std mmap" }
```

actionlint (1.7.7)

label "macos-15-intel" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2022", "windows-2019", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "ubuntu-20.04", "macos-latest", "macos-latest-xl", "macos-latest-xlarge", "macos-latest-large", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xl", "macos-14-xlarge", "macos-14-large", "macos-14", "macos-13-xl", "macos-13-xlarge", "macos-13-large", "macos-13", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)